FlowersStoreTests/TestHelpers/VerifyMethodWrapper/VerifyMethodVoid.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  VerifyMethodVoid.swift
+//  ShoestockTests
+//
+//  Created by Rafael Felipe on 28/10/21.
+//  Copyright © 2021 Netshoes. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+/// A generic property wrapper to create unit spy verification property
+///
+/// Use `VerifyMethodVoid` when you need spy a method with without arguments
+///
+/// To spy you should call `call()` inside the method that will be checked
+///
+/// * verify a method without arguments
+///
+///       @VerifyMethodVoid private(set) var verifyMethodFooBar
+///
+///       func methodFoo() {
+///            verifyMethodFooBar.call()
+///       }
+///
+/// More detail, please check the documentation on Confluence (https://magazine.atlassian.net/l/c/3mk7uZ1d) page
+///
+@propertyWrapper struct VerifyMethodVoid {
```

Comment:
<!--
  0 failure: 
  1 warning:  Struct 'VerifyMet...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStoreTests/TestHelpers/VerifyMethodWrapper/VerifyMethodVoid.swift;
  Line: 28;
-->

- :warning: Struct 'VerifyMethodVoid' is unused